### PR TITLE
Us 67 vu

### PR DESCRIPTION
**Pull request by vupham129**
Please review my codes for US67.

### CODE REVIEW

File: src/test/java/com/vytrack/test/US_67.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+    }
+
+    @DataProvider(name="userSalesManagerProvider")
+    public Object[][] provideData2(){
+        return new Object[][]{
+                {ConfigurationReader.getProperty("usernameSalesManager1"), ConfigurationReader.getProperty("password")},
+                {ConfigurationReader.getProperty("usernameSalesManager2"), ConfigurationReader.getProperty("password")},
+                {ConfigurationReader.getProperty("usernameSalesManager3"), ConfigurationReader.getProperty("password")},
+                {ConfigurationReader.getProperty("usernameSalesManager4"), ConfigurationReader.getProperty("password")}
```
Comment (TamerlanHuseynov) [inline src/test/java/com/vytrack/test/US_67.java:52]: You can just write here keywords without calling and Configuration Reader get Property. And password also can be written as keyword.

File: src/test/java/com/vytrack/test/US_67.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        };
+    }
+
+    @Test(dataProvider = "userTruckDriversProvider", priority = 1)
+    public void truckDriversTest(String username, String password) {
+        //AC #1: Users should see three columns on the Vehicle Costs page.
+            //step1:
+        VyTrack_Login.login(Driver.getDriver(), username, password);
```
Comment (TamerlanHuseynov) [inline src/test/java/com/vytrack/test/US_67.java:60]: And here you can import Configuration Reader get property for username and password. Every time it is gonna take data provider keys and return their value for you.

**Review by TamerlanHuseynov — CHANGES_REQUESTED** [review]
Your code looks good besides the data provider. It has long lines which can be written in a shorter way. I have put comments on the exact code lines that have to be changed. Please review your code again and don't hesitate to ask questions.

File: src/test/java/com/vytrack/test/US_67.java (hunk 1 above)
Comment (vupham129) [reply src/test/java/com/vytrack/test/US_67.java:52]: okie. i will take a look and fix it

File: src/test/java/com/vytrack/test/US_67.java (hunk 2 above)
Comment (vupham129) [reply src/test/java/com/vytrack/test/US_67.java]: fixed, Please take a look :)

**Review by MohammedAlshalchi — APPROVED** [review]
It looks good codes but as Aaron said you can use one credential for each

**Review by TamerlanHuseynov — APPROVED** [review]
Well done! It seems you have time to  pick another  not assigned US and work on it :)

**Comment by vupham129** [comment]
@TamerlanHuseynov I can take Azaman's US to work on.

**Comment by vupham129** [comment]
@TamerlanHuseynov I'll work on Maurice's US

**Review by Linderwow — APPROVED** [review]
Great Job. Keep up the good work.

**Review by KatiU — APPROVED** [review]
Great job.Good work.

**Pull request merged**